### ISSUE TITLE
Checkbox accepting null ariaDescribedBy and creating an attribute without value

### ISSUE THREAD

**Issue by AkshanshSrivastava03** [OfficeDev/office-ui-fabric-react#4985]
#### Describe the issue:
When we pass ariaDescribedBy as null in Checkbox, the renderer renders aria-describedby attr without any value causing Keros failure (Accessibility)

#### Actual behavior:
The renderer renders aria-describedby attr without any value causing Keros failure (Accessibility)

#### Expected behavior:
Control should handle that null pass of ariaDescribedBy should not even create an attr.

### If applicable, please provide a codepen repro:

https://codepen.io/AkshanshSrivastava03/pen/wmdEVJ

**Comment by AkshanshSrivastava03** [comment]
@JasonGore 
I am still able to see aria-describedby attr without any value in the DOM if undefined or null gets passed to Checkbox. Can we re-open this bug? 
I am using  "office-ui-fabric-react": "^5.112.1004" version

**Comment by JasonGore** [comment]
I am not seeing the behavior you describe in this [https://codepen.io/jasongore/pen/bxvBNE]CodePen. Only the first Checkbox with a valid `ariaDescribedBy` prop value has the `aria-described-by` attribute. Can you provide a CodePen or modify the one I provided to show the behavior you are seeing? Thanks.
